mailbox/event/event-memory/src/test/java/org/apache/james/mailbox/events/MemoryEventDeadLettersTest.java

```diff
@@ -22,13 +22,14 @@
 import static org.apache.james.mailbox.events.EventDeadLettersContract.FailedEventContract;
 import static org.apache.james.mailbox.events.EventDeadLettersContract.FailedEventsContract;
 import static org.apache.james.mailbox.events.EventDeadLettersContract.GroupsWithFailedEventsContract;
+import static org.apache.james.mailbox.events.EventDeadLettersContract.ContainEventsContract;
 import static org.apache.james.mailbox.events.EventDeadLettersContract.RemoveContract;
 import static org.apache.james.mailbox.events.EventDeadLettersContract.StoreContract;
 
 import org.junit.jupiter.api.BeforeEach;
 
 class MemoryEventDeadLettersTest implements StoreContract, RemoveContract, FailedEventContract, FailedEventsContract,
-    GroupsWithFailedEventsContract {
+    GroupsWithFailedEventsContract, ContainEventsContract {
```

Comment:
Can we have a `AllTests` interface in `EventDeadLettersContract` extending all these sub contracts so that we can only write:

```
class MemoryEventDeadLettersTest implements EventDeadLettersContract.AllTests { ...
```

That way we can easily ensure we forgot none...

Reply:
What about `AllContracts` because the context is in `EventDeadLettersContract`